src/specific/init.c

```diff
@@ -139,7 +139,7 @@ void *game_malloc(int32_t alloc_size, GAMEALLOC_BUFFER buf_index)
     void *result = GameAllocMemPointer;
     GameAllocMemFree -= aligned_size;
     GameAllocMemUsed += aligned_size;
-    GameAllocMemPointer += aligned_size;
+    GameAllocMemPointer = (char *)GameAllocMemPointer + aligned_size;
```

Comment:
Hmm, what do you think about changing this pointer to `char*` in its definition?

Reply:
no issues from Clang either.